### ISSUE TITLE
Self-destruction

### ISSUE THREAD

**Issue by tcf-admin** [tcfev/forDem#108]

**Comment by zanzendegi** [comment]
Self-destruction could be an option, not forced to everyone.

We can also let the users setup two passwords. The second one destroys everything unattended.

**Comment by armantorkzaban** [comment]
I added your idea to the main post.

**Comment by Ver-Nunft** [comment]
couldn't you just nuke other accounts with nth wrong password account destruction 🤔?

**Comment by desputinski** [comment]
Yes, I think that is why the suggestion was, that only a certain wrong password destroys the account. So for example, my correct password ist 123abc. But if I type abc123 instead, boom, account/app/history destroyed. But if I type 123acb or 132abc, nothing happens.
